translator/src/test/java/com/google/devtools/j2objc/gen/StatementGeneratorTest.java

```diff
@@ -132,6 +132,20 @@ public void testCatchTranslation() throws IOException {
     assertEquals("@try {\n;\n}\n @catch (JavaLangException *e) {\n}", result);
   }
 
+  public void testMultiCatchTranslation() throws IOException {
```

Comment:
Good call, I had overlooked `testMutiCatch()` earlier. Updated the pull request but used `assertNotInTranslation()` instead of `assertTranslatedLines()` for the sake of brevity. Do you think that's sufficient?